pkg/forklift-api/webhooks/mutating-webhook/mutators/ovirt-cert-mutator.go

```diff
@@ -69,9 +69,17 @@ func (mutator *OvirtCertMutator) Mutate(ar *admissionv1.AdmissionReview) *admiss
 			return util.ToAdmissionResponseError(err)
 		}
 
+		// Remove all new lines from both certs for the comparison.
+		secretCa := bytes.ReplaceAll(secret.Data["cacert"], []byte{0x0a}, []byte{})
```

Comment:
wouldn't it be more elegant with bytes.TrimSpace?

Reply:
I've tried it initially using `bytes.TrimSpace`, its doesn't work properly when removing spaces, only new lines (0x20 vs 0x0a)

Reply:
ok